rules/windows/defense_evasion_mshta_beacon.toml

```diff
@@ -1,8 +1,8 @@
 [metadata]
 creation_date = "2020/09/02"
-integration = ["endpoint", "windows"]
+integration = ["endpoint", "windows", "sentinel_one_cloud_funnel"]
 maturity = "production"
-updated_date = "2025/03/20"
```

Comment:
FYI - this rule looks like a duplicate of https://github.com/elastic/detection-rules/blob/fbfc696a86d3ec826fbd13fb8957d294f155dc17/rules/windows/defense_evasion_network_connection_from_windows_binary.toml#L135 but worth keep it separated as is (for difference noise level as well as its one of the top abused lolbin this last year)